src/components/hoc/withExtendedTouchable.tsx

```diff
@@ -53,15 +53,15 @@ export function withExtendedTouchable<T extends TouchableOpacityProps | Touchabl
         // Async onPress. Always show busy form until promise resolves
         if (typeof result?.then === 'function') {
           setIsAwaiting(true)
-          result.catch(e => showError(e)).finally(() => setIsAwaiting(false))
+          result.catch(e => showDevError(e)).finally(() => setIsAwaiting(false))
         }
 
         // Also layer on debounce state toggling if enabled
         if (debounce) {
           timeoutRef.current = setTimeout(() => setIsDebouncing(false), DEBOUNCE_DELAY)
         }
       } catch (err) {
-        showError(err)
+        showDevError(err)
```

Comment:
Same here: The general principle is that we should always show errors in *response to user actions*.

We *only* want to hide errors that happen during automatic background operations. Things like updating exchange rates, fetching staking status, syncing blockchains, downloading message cards on boot, etc... are all candidates for being hidden.